index.html

```diff
@@ -30,87 +36,72 @@ <h6 class="a">LET’S CONNECT</h6>
           <li class="i"><a href="#"><img src="images/medium.png" alt=""></a></li>
         </ul>
       </section>
+<!-- WORKS -->
       <section class="All-section" >
```

Comment:
- [ ] Your works section should be aligned in the center of the screen as it is in the Figma template. Kindly fix that.
![image](https://user-images.githubusercontent.com/61919617/172725091-48ce30c5-492a-4451-b8f1-c675ba95d47a.png)
![image](https://user-images.githubusercontent.com/61919617/172725166-5d3f1997-423e-4797-b32b-81db903df4b9.png)